src/com/insightdatascience/onpoint/LICENSE.txt

```diff
@@ -0,0 +1,17 @@
+MIT License
+Copyright (c) 2018 YOUR NAME
```

Comment:
Change "YOUR NAME" to your own name and "2018" to "2019".

Reply:
Already has 'License.text' outside the onpoint folder, so I deleted this one.